protocols/randhound/randhound.go

```diff
@@ -844,6 +857,14 @@ func (rh *RandHound) handleR2(r2 WR2) error {
 	}
 
 	if proceed && !rh.SecretReady {
+
+		// XXX: dummy operation that simulates aggregation of the server responses in the 3-round version of RandHound
```

Comment:
Since we're onto that path, maybe adding a `time.Sleep(RTT)` should definitely make a better approx.